src/dropLine.js

```diff
@@ -53,6 +53,14 @@ export default (config, xScale) =>
             )
             .call(drop(config, xScale));
 
+        drops
+            .append('rect')
+            .attr('x', 0)
+            .attr('y', -config.line.height / 2)
+            .attr('width', 1)
```

Comment:
Add a comment to explain the purpose of this rectangle. A width of 1 is indeed quite weird.